python/cuml/manifold/simpl_set.pyx

```diff
@@ -138,6 +139,14 @@ def fuzzy_simplicial_set(X,
     umap_params.deterministic = <bool> deterministic
     umap_params.set_op_mix_ratio = <float> set_op_mix_ratio
     umap_params.local_connectivity = <float> local_connectivity
+    try:
+        umap_params.metric = metric_parsing[metric.lower()]
```

Comment:
I noticed that the documentation currently states that the `metric` and `metric_kwds` are unused. Can you fix that, please?